server_benchmark.py

```diff
@@ -91,9 +93,26 @@ def benchmarkOneQuery(request, repeatLimit=3, pageLimit=3):
 if __name__ == '__main__':
     parser = argparse.ArgumentParser(
         description="GA4GH reference server benchmark")
+    parser.add_argument('--profile', default='none',
+                        choices=['none', 'heap'],
+                        help='"heap" runs a heap profiler '
+                             'once inside the backend')
+    parser.add_argument('--repeatLimit', type=int, default=3, metavar='N',
+                        help='how many times to run each test case '
+                             '(default: %(default)s)')
+    parser.add_argument('--pageLimit', type=int, default=3, metavar='N',
+                        help='how many pages (max) to load '
+                             'from each test case (default: %(default)s)')
+    args = vars(parser.parse_args())
```

Comment:
Good catch! I'd been misled by skimming documentation, and it's much cleaner without the vars() call.